### ISSUE TITLE
@types/recharts/index has no exported member LabelList

### ISSUE THREAD

**Issue by cpetter** [recharts/recharts#1327]
Hello

I use recharts to display some BarChart for my project and it works pretty fine.
But I'm not able to use the LabelList in my React.Component

If I try to do 
`import { LabelList } from 'recharts' `

I get **types/recharts/index has no exported member LabelList**

I don't find this LabelList in the index.d.ts of @types/recharts so I suppose this is the problem ?

Is there any other way to import this LabelList
The recharts version is the 1.0.20 and before I got 1.0.16 and it didn't work as well.

And if I try to use `const { LabelList } = Recharts` like in examples, I get
**Cannot find name Recharts**

I'm using VSCode to develop my React project
Many thanks for your help

Cédric

**Comment by TeemuKoivisto** [comment]
Not to chide you or anything but I created a PR almost two weeks ago https://github.com/recharts/recharts/issues/1317 to which I replied 4 days ago that I've made a PR which is just waiting to be approved. I know it's fun creating issues but often it helps maintainers when you do a little research before leaving issues that might remain open for undetermined periods of time.

As of today it was merged and the LabelList is included in the `@types/recharts 1.0.21`
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26203

To be honest I wasn't completely sure I had the typings right but well I guess if maintainers approved it they were.
